tests/CLICommandTest.php

```
@@ -59,6 +59,40 @@ public function tearDown(): void {
 
+	/**
+	 * @param string     $memory_limit
+	 * @param int|string $os_virtual_memory_kb
+	 * @return void
+	 */
+	private function call_check_memory_limit( $memory_limit, $os_virtual_memory_kb = 'n/a' ) {
+		$cli_command = new CLI_Command();
+		$method      = new \ReflectionMethod( $cli_command, 'check_memory_limit' );
+		if ( PHP_VERSION_ID < 80100 ) {
+			// @phpstan-ignore method.deprecated
+			$method->setAccessible( true );
```

PHPStan (2.2.7)

[error] No error with identifier method.deprecated is reported on line 72.


(ignore.unmatchedIdentifier)